### PR TITLE
ci: add common actions, add parent to pom

### PR DESCRIPTION
**Pull request by genryxy**
Part of #2 
Added common GitHub actions, added parent to pom file

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      - "dependencies"
+      - "bot"
+    commit-message:
+      prefix: "deps"
+    assignees:
+      - "genryxy"
+    reviewers:
+      - "genryxy"
+    schedule:
+      interval: "daily"
```
Comment (olenagerasimova) [inline .github/dependabot.yml:18]: @genryxy is not it too often? I'd suggest weekly, if it's possible

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy thanks, just one suggestion

**Comment by genryxy** [comment]
@olenagerasimova thanks, changed to once a week

**Review by olenagerasimova — APPROVED** [review]
@genryxy thanks

**Pull request merged**